patterns/1-initial/extensions-to-manage-contributions-at-scale.md

```diff
@@ -0,0 +1,83 @@
+## Title
+
+Extensions to Manage Contributions at Scale
+
+## Patlet
+
+An InnerSource project is receiving too many contributions, making maintenance difficult and resulting in a large code review backlog or premature rejection of new feature contributions. By offering an extension mechanism outside of the core project, the maintainers enable contributors to extend project capabilities with minimal code, thus encouraging innovation and experimentation. This would reduce the maintenance overhead on the core project, allowing it to contain only the strategic features that have been widely adopted, tested and validated. It would also ensure faster release of new features to the community, reducing the cost to harden a feature until it has proven its utility.
```

Comment:
```suggestion
An InnerSource project is receiving too many contributions, making maintenance difficult and resulting in a large code review backlog or premature rejection of new feature contributions. By offering an extension mechanism outside of the core project, the maintainers enable scaling of project capabilities with minimal cost and maintenance overhead. 
```

Reply:
How about this patlet @spier @alex-jw-brooks @gabe-l-hart ?

Reply:
I like this simpler version